pandas/core/algorithms.py

```diff
@@ -289,9 +289,9 @@ def unique(values):
     Returns
     -------
     unique values.
-      If the input is an Index, the return is an Index
-      If the input is a Categorical dtype, the return is a Categorical
-      If the input is a Series/ndarray, the return will be an ndarray
+        If the input is an Index, the return is an Index.\n
+        If the input is a Categorical dtype, the return is a Categorical.\n
+        If the input is a Series/ndarray, the return will be an ndarray.
```

Comment:
Did you try using a list (adding a dash before each), instead of this? If you don't mind rendering the html with it, may be it looks better. Then, you can add the period at the end of each, instead of the `\n`

Reply:
I think the issue is (if I remember correctly) the dash causes `RT04` error and `RT05` will skip if there is a dash. I rendered `pandas.Series.agg` and it looks fine. But let me know if you think we should use a dash.